packages/app/src/app/pages/Sandbox/Editor/Header/index.tsx

```diff
@@ -217,12 +240,16 @@ const Header = ({ store, signals, zenMode }: Props) => {
             likeCount={store.editor.currentSandbox.likeCount}
           />
         )}
-        <ShareButton
-          style={{ fontSize: '.75rem', margin: '0 1rem' }}
-          signals={signals}
-          secondary={!sandbox.owned}
-          store={store}
-        />
+
+        {store.user.curatorAt && (
```

Comment:
It think there needds to be a nullcheck here to be sure.

```suggestion
        {store.user && store.user.curatorAt && (
```

Reply:
👍